images/hook/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/dumb-init /bin/sh
+set -e
+
+if [ $1 = "update" ]; then
+    echo "Starting update, changeset: $RIG_CHANGESET"
+    echo "Deleting old replication controller rc/log-forwarder"
+    rig delete rc/log-collector --resource-namespace=kube-system
+    echo "Creating or updating resources"
+    rig upsert -f /var/lib/gravity/resources/resources.yaml --debug
+    echo "Checking status"
+    rig status $RIG_CHANGESET --retry-attempts=120 --retry-period=1s --debug
+    echo "Freezing"
+    rig freeze
+else
```

Comment:
Okay, that makes a bit more sense. What's responsible for calling rollback hook? Automatically called by platform if update hook fails?